assets/constants.js

```diff
@@ -40,6 +40,52 @@ var constants = {
   // The default language code for an exploration.
   "DEFAULT_LANGUAGE_CODE": "en",
 
+  "DEFAULT_OBJECT_VALUES": {
```

Comment:
@esyang97, This is entirely a duplicate value of `extensions/objects/object_defaults.json` right? If yes, I think we should delete that file (if we are going to follow this pattern.)

@seanlip, will that be fine to delete `extensions/objects/object_defaults.json` and move it's content here?

Reply:
My initial solution involved an ajax get request to `extensions/objects/object_defaults.json` that I was unable to get to work.
https://github.com/oppia/oppia/pull/6636/commits/7f35db52d86077db569ff574ba23016a186d8495
This may be the optimal solution. Would I be able to get some assistance in this?

Also, `extensions/objects/object_defaults.json` is used in `core/domain/obj_services.py`

Reply:
@DubeySandeep this is really a question for @vojtechjelinek, he owns the build process. Cc-ing him here.

Reply:
@esyang97, can you modify  `obj_services.py` to use the constants.DEFAULT_OBJECT_VALUES?  If yes then I think this approach looks good to me!

Reply:
as `constants.js` is a .js file, I can do the following:

1. replace `OBJECT_DEFAULT_VALUES_FILE_PATH` in `feconf.py` with the actual dictionary

2. alter `obj_services.py` and all files that use `get_default_object_values()` which are only test files `extensions/interactions/base_test.py` and `core/domain/obj_services_test.py`

3. delete `extensions/objects/object_defaults.json`

What do you think?

Reply:
@esyang97, you can use constants.js values directly constants.<field-name> (using dot notation this functionality is defined in constants.py). So you can remove OBJECT_DEFAULT_VALUES_FILE_PATH from feconf.py.

Reply:
I think it is okay to move the `object_defaults.json` to `constants.js`